internal/pkg/describe/stack.go

```diff
@@ -52,3 +53,14 @@ func (d *stackDescriber) StackResources(stackName string) ([]*cloudformation.Sta
 	}
 	return out.StackResources, nil
 }
+
+// Template returns the CloudFormation template of the stack.
+func (d *stackDescriber) Template(stackName string) (string, error) {
+	out, err := d.stackDescribers.GetTemplate(&cloudformation.GetTemplateInput{
```

Comment:
What do you think of using [GetTemplateSummary](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_GetTemplateSummary.html) to retrieve the metadata 🤔

Reply:
😮 neat! I didn't know this API existed, it simplified a bunch of stuff. Done!